airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,359 @@
+
+This table has many-to-one relationship with the `WORKSPACE` table. One workspace can have multiple notifications.
+
+This is the base table for notifications. Each notification type will have its own child table (e.g. `SLACK_NOTIFICATION`).
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| notification_type | varchar(50) | yes | | Index. String enum: "slack". |
+| send_on_success | boolean | | false | index |
+| send_on_failure | boolean | | false | index |
+
+### `SLACK_WORKSPACE_NOTIFICATION`
+
+YAML reference:
+- [SlackNotificationConfiguration](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SlackNotificationConfiguration.yaml)
+
+This table has one-to-one relationship with the `WORKSPACE_NOTIFICATION` table. Each `SLACK_WORKSPACE_NOTIFICATION` belongs to one `WORKSPACE_NOTIFICATION`.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| notification_id | char(36) | yes | | Foreign key. Unique index to enforce 1:1 mapping. |
+| webhook | varchar(2048) | yes | | |
+
+### `SOURCE_DEFINITION`
+
+YAML reference:
+- [StandardSourceDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(255) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(50) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `SOURCE_CONNECTION`
+
+YAML reference:
+- [SourceConnection](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SourceConnection.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | |
+| source_definition_id | char(36) | yes | | Foreign key. Index. |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| configuration | jsonb | yes | | Must be a valid json string. |
+| tombstone | boolean | | false | index | |
+
+### `DESTINATION_DEFINITON`
+
+YAML reference:
+- [StandardDestinationDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardDestinationDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(36) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(63) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `DESTINATION_CONNECTION`
```

Comment:
same comment as `SOURCE_CONNECTION`

Reply:
If we wanted to combine this into the same table as SOURCE_CONNECTION--based on the same reasoning you were using source definition and destination definition we could. The only thing that will be a bit awkward is the name.